bamboo/unit-tests.sh

```diff
@@ -5,8 +5,8 @@ set -ex
 docker ps -a ## Show running containers for output logs
 
 # Run unit tests (excluding integration/api tests)
-docker exec -i ${container_id}\_build_env_1 /bin/bash -c 'cd /source/cumulus; nyc ./node_modules/.bin/lerna run test --ignore @cumulus/api --ignore cumulus-integration-tests'
+docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR; nyc ./node_modules/.bin/lerna run test --ignore @cumulus/api --ignore cumulus-integration-tests"
 # Run api tests
-docker exec -i ${container_id}\_build_env_1 /bin/bash -c 'cd /source/cumulus/packages/api; npm run test-coverage'
+docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR/packages/api; npm run test-coverage"
 # Report combined code coverage
-docker exec -i ${container_id}\_build_env_1 /bin/bash -c 'cd /source; mkdir .nyc_output; cp -a /source/cumulus/.nyc_output/. /source/.nyc_output/; cp -a /source/cumulus/packages/api/.nyc_output/. /source/.nyc_output/; nyc report;'
+docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd /source; mkdir .nyc_output; cp -a $UNIT_TEST_BUILD_DIR/.nyc_output/. /source/.nyc_output/; cp -a $BUILD_DIR/packages/api/.nyc_output/. /source/.nyc_output/; nyc report;"
```

Comment:
Should `$BUILD_DIR` also be `$UNIT_TEST_BUILD_DIR`?

Reply:
Yes, will update.

Reply:
b534a3a70